test/unit/utils-commands-tab.spec.js

```diff
@@ -0,0 +1,400 @@
+import {describe, expect, it} from 'vitest';
+
+import {
+  adjustParamValueType,
+  deepFilterEmpty,
+  extractParamsFromCommandPath,
+  filterMethodPairs,
+  transformCommandsMap,
+  transformExecMethodsMap,
+} from '../../app/common/renderer/utils/commands-tab.js';
+
+describe('utils/commands-tab.js', function () {
+  describe('#adjustParamValueType', function () {
+    it('should detect the correct type for common examples', function () {
```

Comment:
you may use the following approach to avoid unnecessary repetition in parametrized tests:

```
const cases = [
  ['test', 'test'],
  ['109', 109],
  ['true', true],
  ['false', false],
  ['null', null],
  ['[1, 2, 3]', [1, 2, 3]],
  ['{"a":1}', { a: 1 }],
];

cases.forEach(([input, expected]) => {
  it(`should detect correct type for input "${input}"`, function () {
    expect(adjustParamValueType(input)).to.eql(expected);
  });
});
```